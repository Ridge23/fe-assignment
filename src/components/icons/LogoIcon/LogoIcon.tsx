import React from 'react';

import './sass/logo-icon.scss';

export default function LogoIcon(): JSX.Element {
    return <i className="logo-icon" />
}