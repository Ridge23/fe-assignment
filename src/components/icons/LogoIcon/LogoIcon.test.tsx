import React from 'react';
import { shallow } from 'enzyme';

import LogoIcon from './LogoIcon'

describe('LogoIcon', () => {
  it('should render correctly', () => {
    const component = shallow(<LogoIcon />);

    expect(component).toMatchSnapshot();
    expect(component.hasClass('logo-icon')).toBe(true);
  });
});