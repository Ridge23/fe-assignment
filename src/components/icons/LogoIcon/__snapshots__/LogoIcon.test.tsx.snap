// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoIcon should render correctly 1`] = `
<i
  className="logo-icon"
/>
`;
